### PR TITLE
send email via mailgun api.

### PR DESCRIPTION
**Pull request by felixguerrero12**
This pull request is submitted to add email via mailgun api. Beautified the manifest.json file, and added the mailgun section as 008.

### CODE REVIEW

**Comment by felixguerrero12** [comment]
This code will need to be modified to call from the locker or even the environment variable. The goal is to have a static e-mail service to enable the administrator send e-mails using api.

**Comment by felixguerrero12** [comment]
I am a shit git user. This includes both the s3, and mailgun. going to see if I can fix this up.

File: operators/s3.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        access needed: s3:ListAllMyBuckets
+        """
+        client = self.connect()
+        for bucket in client.buckets.all():
+            print(bucket.name)
+
+
+if __name__ == "__main__":
+    out = code(aws_access_key_id, aws_secret_access_key, aws_region)
+    out.list_buckets()
```
Comment (bmarsh9) [inline operators/s3.py:43]: @felixguerrero12 
Unfortunately Spate won't be able to support this format (I wish). Your other operator [here](https://github.com/bmarsh9/spate-operators/pull/1/files#diff-aee348fe6b9fac5e95a99a0a0a5f6cf59b4225bf3c0cc45099fb862d733f5489) is the correct format

**Review by bmarsh9 — CHANGES_REQUESTED** [review]
Everything looks good except for the format in [s3.py](https://github.com/bmarsh9/spate-operators/pull/1/files#diff-2ac2fb13e9512b5e973588472978a2a73bd348854e78b3455fb773cf14e92c0e)

**Pull request closed without merging**